### ISSUE TITLE
SVG files that are bitmap blobs

### ISSUE THREAD

**Issue by mroth** [openlawteam/openlaw-website#21]
A number of SVG files are stored internally as binary bitmap images, rather than vector graphics, resulting in large file sizes:

- `bg.svg` (183kb)
- `build.svg` (158kb)
- `community.svg` (214kb)

Looks like the same for the following SVGs currently set in JSX have the same issue:

```
$ cd /src/components/svg; grep -l "data:image/png;base64" *.{js,jsx}                
BlueBoxesSVG.jsx
BuildSVG.jsx
CommunitySVG.jsx
ContractsModelSVG.jsx
DarkBlueBoxSVG.jsx
LightBlueBoxSVG.jsx
```

Different but related, but there are also  a number of images that look like they were originally vector based, but only have PNG in the src currently, so having a SVG vector version could make things smaller:

- `Lock`
- `Machine`
- `File-pattern`
- `Fun-bunch-*`

I know we're still iterating on design, but figured I'd compile the list while I noticed it before I forgot.

xoxo,
person who loads websites on slow plane wifi far too often ✈️ 🤣

**Comment by nostalgic** [comment]
Ok, I will look into this and provide clean .svgs for all assets

**Comment by jtrein** [comment]
Fixed in #50
